packages/vite/src/node/plugins/css.ts

```diff
@@ -288,23 +288,26 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         if (isDirectCSSRequest(id)) {
           return css
         } else {
+          const cssStr = JSON.stringify(css)
           // server only
           if (options?.ssr) {
-            return modulesCode || `export default ${JSON.stringify(css)}`
+            return modulesCode || `export default ${cssStr}`
           }
           if (inlined) {
-            return `export default ${JSON.stringify(css)}`
+            return `export default ${cssStr}`
           }
+          const idStr = JSON.stringify(id)
           return [
-            `import { updateStyle, removeStyle } from ${JSON.stringify(
+            `import { updateStyle as __vite__updateStyle, removeStyle as __vite__removeStyle } from ${JSON.stringify(
               path.posix.join(config.base, CLIENT_PUBLIC_PATH)
             )}`,
-            `const id = ${JSON.stringify(id)}`,
-            `const css = ${JSON.stringify(css)}`,
-            `updateStyle(id, css)`,
+            `__vite__updateStyle(${idStr}, ${cssStr})`,
             // css modules exports change on edit so it can't self accept
-            `${modulesCode || `import.meta.hot.accept()\nexport default css`}`,
-            `import.meta.hot.prune(() => removeStyle(id))`
+            `${
+              modulesCode ||
+              `import.meta.hot.accept()\nexport default ${cssStr}`
+            }`,
+            `import.meta.hot.prune(() => __vite__removeStyle(${idStr}))`
```

Comment:
If I'm reading this correctly, this would double the `cssStr` for a CSS module, which could cause longer fetch time. I wonder since we're already prefixing import names with `__vite__`, we can do so for the `id` and `css` variables too? Something like:

```js
const __vite__id = "..."
const __vite__css = "..."
__vite__updateStyle(__vite__id, __vite__css)
```

Reply:
I have changed `cssStr` to use `__vite__` prefix, thanks for your review.